api/maven-api-core/src/main/java/org/apache/maven/api/JavaPathType.java

```diff
@@ -63,16 +75,17 @@ public enum JavaPathType implements PathType {
      *       class-path.</li>
      * </ul>
      */
-    CLASSES("--class-path"),
+    CLASSES(StandardLocation.CLASS_PATH, "--class-path"),
```

Comment:
Is the `option` string still useful ? That looks really redundant and specific to tool implementations.  Can we only keep the `StandardLocation` ?

Reply:
The presence or not of this string depends on the [comment below](https://github.com/apache/maven/pull/1501#discussion_r1590432486).